src/rust/dbv2/migrations/2023-09-04-113144_postgrest/up.sql

```diff
@@ -0,0 +1,29 @@
+-- Your SQL goes here
+CREATE ROLE web_anon nologin;
+
+
+CREATE SCHEMA api;
+
+
+CREATE VIEW
+  api.market_registration_events AS
+SELECT
+  *
+FROM
+  market_registration_events;
+
+
+GRANT usage ON SCHEMA api TO web_anon;
+
+
+GRANT
```

Comment:
Add to this:

```
REVOKE SELECT ON ALL TABLES IN SCHEMA public FROM anon;
REVOKE USAGE ON SCHEMA public FROM anon;
DROP ROLE anon;
```

This is the "down" part of the anon role I added, to be removed here.

Reply:
I removed the part where `anon` is added, so there's no more need for this. I know we discussed not changing old migrations, but this seemed cleaner and since there's no deployment yet, it has no consequences.